server/journeys/screen/confirmation/controller.ts

```diff
@@ -11,6 +11,7 @@ export class ConfirmationController {
   }
 
   private mapOutcomeTypeToStatus(outcomeType: ReferenceData) {
+    // TODO: read computed CSIP status from backend instead
```

Comment:
Might need clarification on this but I thought the decision was made to keep these separate (and the backend had more status types than we show here)

Reply:
i think you're right that the backend will have more status types than covered by this controller. but the logic should be the same. it's just that backend will cover status types applicable to further down the workflow. up to Screen a Referral, they should have the same logic.